CRM/CiviAwards/Form/AwardReview.php

```diff
@@ -13,7 +15,7 @@ class CRM_CiviAwards_Form_AwardReview extends CRM_Core_Form {
   /**
    * URL for accessing review form from SSP.
    */
-  const SSP_REVIEW_URL = 'civicrm/award-review-ssp';
+  const SSP_REVIEW_URL = 'civicrm/ssp/awardreview';
```

Comment:
Not sure if we change this to `civicrm/ssp/award-review` for better readability. Leave it upto you.

Reply:
I think this is fine because it's similar to the one for civicrm.